patches/@material+mwc-select+0.27.0.patch

```diff
@@ -1,23 +1,25 @@
 diff --git a/node_modules/@material/mwc-select/mwc-select-base.js b/node_modules/@material/mwc-select/mwc-select-base.js
-index 66a2c20..6914121 100644
+index 66a2c20..efcbb8e 100644
 --- a/node_modules/@material/mwc-select/mwc-select-base.js
 +++ b/node_modules/@material/mwc-select/mwc-select-base.js
-@@ -128,6 +128,14 @@ export class SelectBase extends FormElement {
+@@ -128,6 +128,16 @@ export class SelectBase extends FormElement {
          const labelledby = !!this.label ? 'label' : undefined;
          const describedby = this.shouldRenderHelperText ? 'helper-text' : undefined;
          return html `
 +      <style>
 +        .mdc-select__anchor {
 +            min-width: var(--mdc-select-min-width, 200px);
++            max-width: var(--mwc-select-max-width, 100%);
 +        }
 +        .mdc-select--filled {
 +            min-width: var(--mdc-select-min-width, 200px);
++            max-width: var(--mwc-select-max-width, 100%);
 +        }
 +      </style>
        <div
            class="mdc-select ${classMap(classes)}">
          <input
-@@ -247,7 +255,7 @@ export class SelectBase extends FormElement {
+@@ -247,7 +257,7 @@ export class SelectBase extends FormElement {
          if (!this.icon) {
              return nothing;
          }
```

Comment:
Without this patch file, it works properly with `backend-ai-data-view.ts` changes. Is this paths is required? @agatha197

Reply:
You are right. I reverted it.